app/forms/idv/api_image_upload_form.rb

```diff
@@ -94,16 +88,24 @@ def throttled_else_increment
     end
 
     def validate_images
-      IMAGE_KEYS.each do |image_key|
-        validate_image(image_key) if params[image_key]
-      end
+      errors.add(:front, t('doc_auth.errors.not_a_file')) if front.is_a? URI::InvalidURIError
+      errors.add(:back, t('doc_auth.errors.not_a_file')) if back.is_a? URI::InvalidURIError
+      errors.add(:selfie, t('doc_auth.errors.not_a_file')) if selfie.is_a? URI::InvalidURIError
     end
 
-    def validate_image(image_key)
-      file = params[image_key]
-
-      return if file.respond_to?(:read)
-      errors.add(image_key, t('doc_auth.errors.not_a_file'))
+    def as_readable(value)
+      @readable ||= {}
```

Comment:
I'd consider setting this in the initializer

Reply:
> I'd consider setting this in the initializer

Updated in e289f5a0b.